dockers/docker-database/supervisord.conf

```diff
@@ -3,10 +3,19 @@ logfile_maxbytes=1MB
 logfile_backups=2
 nodaemon=true
 
-[program:redis-server]
-command=/usr/bin/redis-server /etc/redis/redis.conf
+[program:start.sh]
+command=/usr/bin/start.sh
```

Comment:
I do not why start redis-server inside start.sh, why can you make start.sh independent of redis-server start. maybe rename it as configdb-load.sh?